glean-core/src/core/mod.rs

```diff
@@ -758,19 +758,40 @@ impl Glean {
             .get_value(self, None)
     }
 
-    /// Set configuration to override the default metric enabled/disabled state, typically from a
+    /// Set configuration to override the default state, typically initiated from a
     /// remote_settings experiment or rollout
```

Comment:
"override the default state"... of what? And how?

Reply:
This may eventually grow to encompass other settings, I made this comment generic and non-specific to allow for that and not create another place that needed kept in sync. The RemoteSettingsConfig seemed a better place to document the what (if not the how) a little closer to the source.